Multi-module projects do not work(?)

Try building [BoneJ2](https://github.com/bonej-org/BoneJ2) with an `imagej.app.directory` set and pom-scijava >= 21.0.0. (Specifically: imagej-maven-plugin 0.7.0 rather than 0.6.0.) The build fails with the following error:
```
[ERROR] Failed to execute goal net.imagej:imagej-maven-plugin:0.7.0:copy-jars (copy-jars) on project pom-bonej2: Couldn't resolve dependencies for artifact: org.eclipse.aether.resolution.DependencyResolutionException: Could not find artifact org.bonej:pom-bonej2:jar:0.4.0-SNAPSHOT in imagej.public (http://maven.imagej.net/content/groups/public) -> [Help 1]
```
Running `mvn -DskipTests clean install` first does not alleviate the issue.

See also [this conversation on Gitter](https://gitter.im/scijava/scijava-common?at=5b040a581cfca775e12b5df7).

--- a/src/main/java/net/imagej/maven/CopyJarsMojo.java
+++ b/src/main/java/net/imagej/maven/CopyJarsMojo.java
@@ -169,6 +169,7 @@ public class CopyJarsMojo extends AbstractCopyJarsMojo {
 		coordinate.setGroupId(project.getGroupId());
 		coordinate.setArtifactId(project.getArtifactId());
 		coordinate.setVersion(project.getVersion());
+		coordinate.setType(project.getPackaging());
 
 		try {
 			TransformableFilter scopeFilter = ScopeFilter.excluding("system", "provided", "test");

